subsys/usb/device_next/class/bt_hci.c

```diff
@@ -0,0 +1,491 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
```

Comment:
Did Intel contribute to this file?

Reply:
This is porting of subsys/usb/device/class/bluetooth.c to new experimental USB device support. My fault, I should mention it in commit message.